addons/sale_stock/tests/test_sale_stock.py

```diff
@@ -32,10 +32,11 @@ def _get_new_sale_order(self, amount=10.0, product=False):
             'partner_invoice_id': self.partner_a.id,
             'partner_shipping_id': self.partner_a.id,
             'order_line': [(0, 0, {
-                'name': product.name,
                 'product_id': product.id,
                 'product_uom_qty': amount,
-                'price_unit': product.list_price})
+                'price_unit': product.list_price,
```

Comment:
You can take the opportunity to update create line to use the command instead :)
```python
            'order_line': [Command.create({
```